packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -0,0 +1,66 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+import { TreeItem } from "./tree-item";
+
+export const TreeItemTemplate = html<TreeItem>`
+    <template
+        role="treeitem"
+        tabindex="${x => (x.disabled ? null : x.focusable ? 0 : -1)}"
+        class="${x => (x.expanded ? "expanded" : "")} ${x =>
+            x.selected ? "selected" : ""} ${x => (x.nested ? "nested" : "")}
+            ${x => (x.disabled ? "disabled" : "")}"
+        aria-expanded="${x => (x.hasItems ? x.expanded : void 0)}"
+        aria-selected="${x => x.selected}"
+        aria-disabled="${x => x.disabled}"
+        @focus=${(x, c) => x.handleFocus(c.event as FocusEvent)}
+        @blur=${(x, c) => x.handleBlur(c.event as FocusEvent)}
+        @keydown=${(x, c) => x.handleKeyDown(c.event as KeyboardEvent)}
+        ${ref("treeItem")}
+    >
+        <div
+            class="positioning-region"
+            part="positioning-region"
+            @click="${(x, c) => x.handleContainerClick(c.event as MouseEvent)}"
+        >
+            <div class="content-region" part="content-region">
+                ${when(
+                    x => !!x.hasItems,
+                    html<TreeItem>`
+                        <div
+                            aria-hidden="true"
+                            class="expand-collapse-button"
+                            part="expand-collapse-button"
+                            @click="${x => x.handleExpandCollapseButtonClick()}"
+                            ${ref("expandCollapseButton")}
+                        >
+                            <slot name="expand-collapse-glyph">
```

Comment:
If I recall, in accordion we have different slots for the expand and the collapse glyph. I could be wrong, but are we doing this consistently across the components? /cc @chrisdholt

Reply:
In the case of tree-view we rotate that icon whereas accordion displays a different icon, I am now not sure we want two separate slots for this in tree-view.